api/util/responses.go

```diff
@@ -19,6 +20,14 @@ type Meta struct {
 	Error   string `json:"error,omitempty"`
 }
 
+// NextPageReq is the request to get the next page of results
+type NextPageReq struct {
+	Method      string `json:"method"`
+	URL         string `json:"url"`
+	ContentType string `json:"contenttype"`
+	JSONBody    string `json:"jsonbody"`
```

Comment:
json representation should be `contentType`, `jsonBody`. Do we _really_ need the content type field, or can we rely on the convention of "supply content type if you want the non-default"?

Reply:
I think including the ContentType in the response is useful because it makes it incredibly apparent how to call the api. It's not strictly necessary, but if a consumer were to look at the json response and see this:

```
"nextPage": {
  "method": "POST",
  "url": "/list",
  "contentType": "application/json",
  "jsonBody": "{\"offset\":50,\"limit\":25}"
}
```

then that's basically telling the exact `curl` command to use:

```
curl -X POST -H "Content-Type: application/json" -d '{"offset":50,"limit":25}' http://localhost:2503/list
```

which is nice for quality of life.

Reply:
I'm realizing now that I've double-raised this issue in review. Answering here for completeness

This convenience is coming at the cost of wire bloat. This change is de-facto sending a constant `"method":"POST","contentType":"application/json",` per request. As UTF8 that's at least 49 bytes of _required_ overhead that we could just cover with documentation. Coupled with the fact that 99.99% of responses will not be read by humans, I think we need to err on the side of performance.